gabbi/driver.py

```diff
@@ -83,7 +83,7 @@ def build_tests(path, loader, host=None, port=8001, intercept=None,
 
     # Exit immediately if we have no host to access, either via a real host
     # or an intercept.
-    if not bool(host) ^ bool(intercept):
+    if not ((host is not None) ^ bool(intercept)):
```

Comment:
FWIW, this seems easier to grok:

```python
if (host is None) ^ (bool(intercept) is False):
```

(or some variation thereof)

I had to look up `^` meaning XOR, but not sure whether that could be simplified.